src/blob/index/bptree/tests.rs

```diff
@@ -75,7 +79,7 @@ async fn serialize_deserialize_file() {
     let meta = vec![META_VALUE; META_SIZE];
     let findex = BPTreeFileIndex::<KeyType>::from_records(
         &Path::new("/tmp/bptree_index.b"),
-        None,
+        create_io_driver(),
```

Comment:
Same in this whole file: create at the beginning of the function, store to field, pass here its clone